docs/content/docs.md

```diff
@@ -183,6 +183,10 @@ refer to a different directory.  On these, paths without a leading `/`
 will refer to your "home" directory and paths with a leading `/` will
 refer to the root.
 
+Additional leading path separators will be ignored, which means a path
```

Comment:
I'm not sure I want to encourage people to add `//`

At some point (for backends which support it like s3) that might actually be a different result.

Reply:
This was primary reason for the PR in the first place... 🤦  But was really just a question/suggestion, so I totally accept your explanation. 😄